cristin-person/src/test/java/no/unit/nva/cristin/person/employment/create/CreatePersonEmploymentKarateRunner.java

```diff
@@ -1,11 +1,28 @@
 package no.unit.nva.cristin.person.employment.create;
 
 import com.intuit.karate.junit5.Karate;
+import no.unit.nva.cristin.person.employment.ClearCristinTestPersonEmployment;
+import nva.commons.core.Environment;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Tag;
 
+import static org.junit.jupiter.api.Assertions.fail;
+
 @Tag("karateTest")
 public class CreatePersonEmploymentKarateRunner {
 
+    public static final String TEST_PERSON_IDENTIFIER = "CRISTIN_EMPLOYMENT_TEST_PERSON_IDENTIFIER";
+
+    @BeforeAll
+    static void clearTestUserEmployments() {
+        try {
+            String cristinTestUserId = new Environment().readEnv(TEST_PERSON_IDENTIFIER);
+            ClearCristinTestPersonEmployment.clearEmployment(cristinTestUserId);
+        } catch (Exception e) {
+            fail("Error reading environment variable "+TEST_PERSON_IDENTIFIER);
```

Comment:
`clearEmployment` also throws ApiGatewayException

Reply:
Format the line too